integration-tests/local-integration-tests/awesome-xcmp.md

```diff
@@ -1,8 +1,10 @@
-## Readings
+## Overview
+
+Resources which allows to grapsh XCM. General undrestanding how bridges work will help too.
```

Comment:
typo: `grapsh` -> `grasp` .
suggestion: reword the sentence `Resources which helps understanding XCM. `